sdk/storage/azure-storage-common/inc/azure/storage/common/crypt.hpp

```diff
@@ -37,7 +37,6 @@ namespace Azure { namespace Storage {
   };
 
   namespace _internal {
-    std::vector<uint8_t> Sha256(const std::vector<uint8_t>& data);
```

Comment:
In general, making breaking changes to `_internal` is not a good idea as it will inevitably break versioning between dependency updates, even if the user only calls and uses `public` surface area. We should therefore carefully review such changes from a customer impact perspective.

This is a rare example of a removal/breaking change to _internal that happens to be OK, only because:
- When the API **originally** shipped it had no callers within the SDK product code, and continues not to have any (across any version), and
- The API and type is more or less in a leaf node within the dependency graph, which has the added benefit of making the audit of usages easier.

It serves as a good example for how we can go about it :)

FYI @JeffreyRichter, @RickWinter